### PR TITLE
Add col 6 to lg screens

### PR DESCRIPTION
**Pull request by caioertai**
Folks, I think this this looks a lot better now. Had to add some BS columns there.
image

### CODE REVIEW

**Comment by daniel-enqz** [comment]
LOOKS AMAZING PANA

**Review by daniel-enqz — APPROVED** [review]
Great

File: app/views/devise/registrations/new.html.erb [hunk 1]
```diff
@@ -1,28 +1,32 @@
-
 <div class="container">
-  <h2>Sign up</h2>
+  <div class="row justify-content-center">
+    <div class="col-lg-6">
```
Comment (caioertai) [inline app/views/devise/registrations/new.html.erb]: What about md? I think we could use 8 there.

File: app/views/devise/registrations/new.html.erb [hunk 2]
```diff
@@ -1,28 +1,32 @@
...
+                      required: true,
+                      hint: ("#{@minimum_password_length} characters minimum" if @minimum_password_length),
+                      input_html: { autocomplete: "new-password" } %>
+          <%= f.input :password_confirmation,
+                      required: true,
+                      input_html: { autocomplete: "new-password" } %>
+        </div>
+
+        <div class="form-actions">
+          <%= f.button :submit, "Sign up" %>
```
Comment (caioertai) [inline app/views/devise/registrations/new.html.erb]: We should add a btn-primary here. It's white now...

**Review by caioertai — COMMENTED** [review]
Added some style comments.

File: app/views/devise/registrations/new.html.erb (hunk 1 above)
Comment (daniel-enqz) [reply app/views/devise/registrations/new.html.erb]: yes

**Comment by caioertai** [comment]
@daniel-enqz , have another look please.

**Pull request merged**